make/registry.mk

```diff
@@ -0,0 +1,23 @@
+OS_IMAGE_REGISTRY := $(shell oc get route default-route -n openshift-image-registry --template='{{ .spec.host }}' 2>/dev/null || true)
```

Comment:
I'm wondering if we still need this registry.mk as we switched to using quay.io ?